### PR TITLE
Request method change

### PR DESCRIPTION
**Pull request by rex-god**
Change POST to GET calls wherever possible, fixed CI/CD deployment & managed NPM as dependency installer (removed yarn). May conflict while managing two different dependency installer.

### CODE REVIEW

File: src/modules/account/account.controller.ts [hunk 1]
```diff
@@ -20,9 +25,8 @@ export class AccountController {
       },
     },
   })
-  @Post('balance')
+  @Get('balance')
   @Version('1')
-  @HttpCode(200)
   async balance(@Body() balanceCheckDto: BalanceCheckDto): Promise<any> {
```
Comment (codinpsycho) [inline src/modules/account/account.controller.ts:30]: @rex-god Hey, we need to use query params for GET.

File: src/modules/nft/components/read-nft/read-nft.controller.ts [hunk 2]
```diff
@@ -12,9 +12,8 @@ export class ReadNftController {
   constructor(private readNftService: ReadNftService) {}
 
   @ReadAllOpenApi()
-  @Post('read_all')
+  @Get('read_all')
   @Version('1')
-  @HttpCode(200)
   async readAllNfts(@Body() readAllNftDto: ReadAllNftDto): Promise<any> {
```
Comment (codinpsycho) [inline src/modules/nft/components/read-nft/read-nft.controller.ts:17]: Same here, use query params for all GET requests

File: src/modules/nft/components/storage-metadata/storage-metadata.controller.ts [hunk 3]
```diff
@@ -9,14 +16,18 @@ import { StorageMetadataService } from './storage-metadata.service';
 @ApiSecurity('api_key', ['x-api-key'])
 @Controller()
 export class StorageMetadataController {
-  constructor(private readonly storageMetadataService: StorageMetadataService) {}
+  constructor(
```
Comment (codinpsycho) [inline src/modules/nft/components/storage-metadata/storage-metadata.controller.ts:19]: Please don't change indentations.

File: src/modules/account/account.controller.ts (hunk 1 above)
Comment (rex-god) [reply src/modules/account/account.controller.ts:30]: Ok

File: src/modules/nft/components/read-nft/read-nft.controller.ts (hunk 2 above)
Comment (rex-god) [reply src/modules/nft/components/read-nft/read-nft.controller.ts:17]: Ok

File: src/modules/nft/components/storage-metadata/storage-metadata.controller.ts (hunk 3 above)
Comment (rex-god) [reply src/modules/nft/components/storage-metadata/storage-metadata.controller.ts:19]: We should maintain same indentation. Right now my indentation is two `space`.

**Pull request closed without merging**